### ISSUE TITLE
List fields missing 

### ISSUE THREAD

**Issue by wolfman6601** [ibericode/mailchimp-for-wordpress#561]
My website URL: password protected during development.

### The issue
List fled are not pulling through to the form in the plugin. Only form fields are available. Please fix.

### How can we reproduce this issue?
1. Activate plugin on website
2. Go to "Form"
3. No list field. This has been set up in Mail Chimp properly.

### Technical info
* WordPress version: 4.9.7
* MailChimp for WordPress version: 4.2.2
* Relevant plugins in case of a bug:

**Comment by dannyvankooten** [comment]
Hi @wolfman6601,

Please go to **WP Admin > MailChimp for WP** and hit the "Renew Lists" button on that page. This will pull in your latest MailChimp lists with all their fields. By default, the plugin will only do this once every 24 hours in order not to slow down your website.

Cheers!

**Comment by wolfman6601** [comment]
Hi Danny

Thanks so much for your reply.

This is a really good plugin. Can't do without it.

I have worked with this plugin for some time already, on other websites.

However, I spent hours yesterday and I certainly tried renewing list a
number of times.

Unfortunately, this did not do anything in terms of placing those buttons
on the form, for easy insertion into the form.

Luckily, I was able to copy the form design from another website and having
worked with Contact Form 7, it's similar, however, even as of this morning,
the form buttons are absent. They should be there even if greyed out.

There is no problem with the plugin on my other site and I am using the
same template for this one.

Something isn't right!

*_____________________*
Regards
Adam Wolfson
